dev/tests/integration/testsuite/Magento/Customer/Block/Adminhtml/Edit/Tab/NewsletterTest.php

```diff
@@ -74,6 +76,9 @@ public function testRenderingNewsletterBlock()
         );
         $this->assertStringNotContainsString('checked="checked"', $body);
         $this->assertStringContainsString('\u003Cspan\u003ESubscribed to Newsletter\u003C\/span\u003E', $body);
-        $this->assertStringContainsString('\u003ENo Newsletter Found\u003C', $body);
+        $this->assertMatchesRegularExpression(
```

Comment:
Hello,
please help me understand: according to the previous assertion `$this->assertStringNotContainsString('checked="checked"', $body);` that means that the customer is not subscribed, the last assertion should fail if I get it right: the customer should not have any queued newsletter associated. But according to the assertions we expect to find a preview link.

Thanks

Reply:
As an example, the customer can unsubscribe from newsletters but have already received ones. They have to be shown in the grid with sent status. The logic of drawing the subscription checkbox is not connected with the newsletters queue.

Reply:
I get the point on the checkbox, but I don't understand where in the fixtures we set a received newsletter from the customer.

To clarify, I verified the PR and it seems ok, it solves the issue, I want just to check whether we have a false positive test or it's just me :)